src/NinjaTools.FluentMockServer.API/Services/ExpectationService.cs

```diff
@@ -95,62 +84,69 @@ public static HttpResponse FakeResponse()
         }
 
 
-        public static Expectation FakeExpectation() => ExpectationFaker.Generate();
-        public static List<Expectation> FakeExpectations(int count) => ExpectationFaker.Generate(count);
-        
+        public static Expectation FakeExpectation()
+        {
+            return ExpectationFaker.Generate();
+        }
+
+        public static List<Expectation> FakeExpectations(int count)
+        {
+            return ExpectationFaker.Generate(count);
+        }
+
         public static JToken FakeBody()
         {
             Func<IFluentBodyBuilder> factory = () => new FluentBodyBuilder();
-            
+
             var faker = new AutoFaker<JToken>();
             var list = BodyOptions.Select(act =>
             {
                 var builder = factory.Invoke();
                 act(builder);
                 return builder.Build();
             }).ToList();
-            
-            return faker.RuleFor( x => x, fake => fake.Random.Shuffle(list));
+
+            return faker.RuleFor(x => x, fake => fake.Random.Shuffle(list));
         }
 
 
         /// <summary>Gets a random <see cref="HttpRequest" />.</summary>
         /// <returns></returns>
-        public static HttpRequest GetRandomHttpRequest() => HttpRequestFaker.Generate();
-    } 
+        public static HttpRequest GetRandomHttpRequest()
+        {
+            return HttpRequestFaker.Generate();
+        }
+    }
+
     public class ExpectationService
     {
-        private readonly ILogger<ExpectationService> _logger;
         private readonly ExpectationDbContext _context;
-        
+        private readonly ILogger<ExpectationService> _logger;
+
         public ExpectationService(ExpectationDbContext context, ILogger<ExpectationService> logger)
         {
             _logger = logger;
             _context = context;
 //            SeedAsync(5);
         }
-        
+
         public async IAsyncEnumerable<Expectation> FindExpectationsAsync(Func<Expectation, bool> predicate, [EnumeratorCancellation] CancellationToken token = default)
         {
             await foreach (var expectation in _context.Expectations.AsAsyncEnumerable().WithCancellation(token))
-            {
                 if (predicate(expectation))
-                {
                     yield return expectation;
-                }
-            }
         }
 
-        public async Task<List<Expectation>> ToListAsync(CancellationToken token = default) => await _context.Expectations.ToListAsync(token);
-        
+        public async Task<List<Expectation>> ToListAsync(CancellationToken token = default)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4739715)